Description: use packaged lempar.c
 Look for lempar.c under the package directory.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2020-12-04

---

--- a/tool/lemon.c
+++ b/tool/lemon.c
@@ -3673,6 +3673,8 @@ PRIVATE FILE *tplt_open(struct lemon *le
     tpltname = buf;
   }else if( access(templatename,004)==0 ){
     tpltname = templatename;
+  }else if( access("/usr/share/lemon/lempar.c",004)==0 ){
+    tpltname = "/usr/share/lemon/lempar.c";
   }else{
     toFree = tpltname = pathsearch(lemp->argv0,templatename,0);
   }
